building/build-debs/homeworld-admin-tools/resources/postinstall.sh

```diff
@@ -40,17 +40,34 @@ cp /keyclient-*.yaml /target/etc/homeworld/config/
 cp /keyclient-base.yaml /target/etc/homeworld/config/keyclient.yaml
 cp /sshd_config.new /target/etc/ssh/sshd_config
 
-cat >/tmp/token.template <<EOF
-Template: homeworld/asktoken
+generate_token_template () {
+    cat >/tmp/$1.template <<EOF
+Template: homeworld/ask$1
 Type: string
-Description: Enter the bootstrap token for this server.
+Description: $2
 
 Template: homeworld/title
 Type: text
 Description: Configuring keysystem...
 EOF
 
-debconf-loadtemplate homeworld /tmp/token.template
+}
+
+generate_token_template "token" "Enter the bootstrap token for this server."
+generate_token_template "tokeninvalid" "Invalid token! Please check the token for typos."
+debconf-loadtemplate homeworld /tmp/token.template /tmp/tokeninvalid.template
```

Comment:
What I meant was -- can you have token.template and tokeninvalid.template be merged into one template file?

Reply:
Oh, merged into one template file, apparently I can do that.